server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -85,10 +93,55 @@ public Result dashboard(Http.Request request, Optional<String> search, Optional<
             pageInfo.getPageCount(),
             pageInfo.getPage(),
             search,
+            searchDate,
             request,
             messagesApi.preferred(request)));
   }
 
+  @Secure(authorizers = Authorizers.Labels.TI)
+  public Result updateDateOfBirth(Long tiId, Long applicantId, Http.Request request) {
+    Optional<CiviFormProfile> civiformProfile = profileUtils.currentUserProfile(request);
+
+    if (civiformProfile.isEmpty()) {
+      return unauthorized();
+    }
+    Optional<TrustedIntermediaryGroup> trustedIntermediaryGroup =
+        userRepository.getTrustedIntermediaryGroup(civiformProfile.get());
+    if (trustedIntermediaryGroup.isEmpty()) {
+      return notFound();
+    }
+    if (!trustedIntermediaryGroup.get().id.equals(tiId)) {
+      return unauthorized();
+    }
+    Form<UpdateApplicantDOB> form =
+        formFactory.form(UpdateApplicantDOB.class).bindFromRequest(request);
+    if (form.hasErrors()) {
+      return redirectToDashboardWithUpdateDateOfBirthError(form.errors().get(0).message(), form);
+    }
+    if (Strings.isNullOrEmpty(form.get().getDob())) {
```

Comment:
issue (blocking): please validate the DOB for:
- being in the correct format
- that the date is in the past